README.md

```diff
@@ -83,13 +83,13 @@ The socket url must also include the transport suffix. For example, if in the se
 
 ### Connection life-cycle
 
-If `init/1` returns `{:connect, url, initial_state}`, the connection will be established immediately. The connection is established in a separate process, which we call the _transport process_. This process is the immediate child of the socket process. Consequently, all communication takes place concurrently to the socket process. If you handle some Erlang messages in the socket process, you may need to keep track of whether you're connected or not.
+If `init/1` returns `{:connect, url, query_params, initial_state}`, the connection will be established immediately. The connection is established in a separate process, which we call the _transport process_. This process is the immediate child of the socket process. Consequently, all communication takes place concurrently to the socket process. If you handle some Erlang messages in the socket process, you may need to keep track of whether you're connected or not.
```

Comment:
Too many commas? I am not the best person when it comes to the use of commas as I use them too liberally myself. However: I think we can skip the comma after the `{:connect, ...}` and after "... separate process".

And also I would skip some in the sentence "If you handle ~~some~~ Erlang messages ..."

I.e. 

> If `init/1` returns `{:connect, url, query_params, initial_state}` the connection will be established immediately. The connection is established in a separate process which we call the _transport process_. This process is the immediate child of the socket process. As a consequence, all communication takes place concurrently to the socket process. If you handle Erlang messages in the socket process you may need to keep track of whether you're connected or not.